packages/cli/src/commands/server/debugger-ui/index.html

```diff
@@ -282,6 +292,9 @@
     <p>Press <kbd id="shortcut" class="shortcut">⌘⌥I</kbd> to open Developer Tools. Enable <a href="https://stackoverflow.com/a/17324511/232122" target="_blank">Pause On Caught Exceptions</a> for a better debugging experience.</p>
     <p>You may also install <a href="https://github.com/facebook/react-devtools/tree/master/packages/react-devtools" target="_blank">the standalone version of React Developer Tools</a> to inspect the React component hierarchy, their props, and state.</p>
     <p>Status: <span id="status">Loading...</span></p>
+    <button class="reload-btn" onclick="onReloadClicked()">
+      Reload
```

Comment:
maybe
```suggestion
      Reload JS
```
what do you think @Esemesek?

Reply:
Or "Reload JS Bundle"? Or perhaps "Reload App"?

Reply:
I think I'm fine with "Reload app"